doc/src/manual/packages.md

```diff
@@ -3,8 +3,8 @@
 Julia has a built-in package manager for installing add-on functionality written in Julia. It
 can also install external libraries using your operating system's standard system for doing so,
 or by compiling from source. The list of registered Julia packages can be found at [http://pkg.julialang.org](http://pkg.julialang.org).
-All package manager commands are found in the `Pkg` module, included in Julia's `Base`
-install.
+All package manager commands are found in the `Pkg` standard library, it becomes available after using
```

Comment:
Suggestion: "it becomes" --> "which becomes", to avoid a [comma splice](https://en.wikipedia.org/wiki/Comma_splice).